Tennis-scheduler/src/main/java/com/tennis/tennisscheduler/controllers/PersonController.java

```diff
@@ -65,4 +73,15 @@ public ResponseEntity<PersonDto> updatePerson(@PathVariable long id,@RequestBody
         Person person = personService.updatePerson(id, personDtoMapper.fromPersonDtoToPerson(personDto));
         return new ResponseEntity<>(personDtoMapper.fromPersonToPersonDto(person),HttpStatus.OK);
     }
+
+    @PutMapping("/")
+    @PreAuthorize("hasRole('TENNIS_PLAYER')")
+    public ResponseEntity<PersonDto> updatePerson(@RequestBody PersonDto personDto){
```

Comment:
Why new endpoint?

Reply:
Second one is for logged user to edit his personal information and the first one is for admin so he can edit information about any user. 
Maybe we don't need the first one endpoint?